### DISCUSSION TITLE
Doesn't work when invoked via lambda due to lambdas only being able to write to /tmp

### DISCUSSION THREAD

**Discussion by RusseII** [pillargg/youtube-upload#17] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/15036618/129940090-819119c2-14a8-4d50-8aa6-21c10ff57e09.png)

**Comment by chand1012** [comment]
You can specify the secrets file path when you initialize the class, see [here](https://docs.pillar.gg/youtube-upload/YoutubeUploader/#youtube_upload.client.YoutubeUploader.__init__).

**Comment by RusseII** [comment]
LUX! Will give it a shot :)

**Comment by RusseII** [comment, marked as answer]
I had similar issue with authorize even when passing in the path [image:
options = {.png]

Ended up just adding  os.chdir('/tmp')
to the lambda

**Comment by chand1012** [comment]
I'm reopening this since the users of this library should be aware of it, and should be fixed at some point.
